tests/test_start.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python
+from selenium.webdriver.common.by import By
+from contextlib import contextmanager
+import time
+
+@contextmanager
+def get_new_windows(selenium, timeout=2):
+    handles = set()
+    wh_before = selenium.window_handles
+    yield handles
+    time.sleep(timeout)
+    handles.update(set(selenium.window_handles).difference(wh_before))
+
+
+def test_click_appstore(selenium, url):
+    selenium.get(url("apps/apps/home/start.ipynb"))
+    with get_new_windows(selenium) as handles:
```

Comment:
```suggestion
    with get_new_windows(selenium, timeout=5) as handles:
```

I am not sure if 5 seconds is long enough. However, I am sure that 2 seconds are too short, as we don't yet see the apps.

Reply:
This timeout is only concerned for how long to wait for new windows. If we need more time for certain elements to appear (e.g. apps on the app store), then we should either query for a specific element or add an additional sleep delay.

Reply:
Ok. Can you please put the sleep delay then?